.travis.yml

```diff
@@ -0,0 +1,19 @@
+dist: xenial
+# Use the latest Travis images since they are more up to date than the stable release.
+group: edge
+sudo: false
+jobs:
+    include:
+    
+    # Run JS tests
+    - language: node_js
+      # The Node version here must be kept in sync with that in `package.json`.
+      node_js:
+        - "14"
+        - "13"
+        - "12"
+     
+     script:
+        - npm run lint
+        - npm test
+        
```

Comment:
Add -npm build as we can check the integrity of the npm file! 
Also, add pre-commit run --all

Reply:
Made the changes